.github/workflows/process_every_payload.yaml

```diff
@@ -0,0 +1,81 @@
+name: Process and Report - every payload
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
+    paths:
+      - "BIPs/**"
+
+env:
+  PR_NUMBER: ${{ github.event.number }}
+  COMMIT_SHA: ${{ github.event.pull_request.head.sha }}
+  WEB3_INFURA_PROJECT_ID: ${{ secrets.WEB3_INFURA_PROJECT_ID }}
+  ETHERSCAN_TOKEN: ${{ secrets.ETHERSCAN_TOKEN }}
+  ARBISCAN_TOKEN: ${{ secrets.ARBISCAN_TOKEN }}
+  GNOSISSCAN_TOKEN: ${{ secrets.GNOSISSCAN_TOKEN }}
+  POLYGONSCAN_TOKEN: ${{ secrets.POLYGONSCAN_TOKEN }}
+  OPTIMISMSCAN_TOKEN: ${{ secrets.OPTIMISMSCAN_TOKEN }}
+  ZKEVMSCAN_TOKEN: ${{ secrets.ZKEVMSCAN_TOKEN }}
+
+jobs:
```

Comment:
I would advise against merging both actions into one. Failure in one of them will fail entire pipeline. Better to have them as one

Reply:
I was thinking the opposite.  We don't want PRs to get merged unless all this stuff actually finishes in an expected way, and it all relates to each other.

Before lots of tests ended red/bad even if they didn't need to run.

We could use multiple jobs as well, but looking more into github actions there's a lot of benefits to keeping this stuff in one file.  Let's think about that more.